backend/core/database.py

```diff
@@ -1523,7 +1523,7 @@ def _update_bgp_updates(self):
                         "LEFT JOIN bgp_updates AS wit ON (hij.key=ANY(wit.hijack_key))) WHERE "
                         "ann.timestamp >= data.v3 AND wit.timestamp >= data.v3 AND "
                         "ann.type='A' AND wit.prefix=ann.prefix AND wit.peer_asn=ann.peer_asn AND wit.type='W' "
-                        "ORDER BY wit_time DESC LIMIT 1) AS witann WHERE witann.wit_time < witann.ann_time) "
+                        "ORDER BY wit_time DESC, hij.key LIMIT 1) AS witann WHERE witann.wit_time < witann.ann_time) "
```

Comment:
what is the perf boost that this entails though? Also is it logically correct?

Reply:
The query goes from 5sec to 20ms.
The link on the PR describes what is the reason this happens